### DISCUSSION TITLE
Initial first-time user thought: What's here?

### DISCUSSION THREAD

**Discussion by lionel** [finnix/finnix#44] [Ideas]
I'm trying to get an idea of what packages/commands are installed above and beyond a typical Debian-based distribution, particularly relating to system maintenance, recovery, and testing. Didn't see anything via a quick perusal of [finnix.org](finnix.org) or these repos.

An auto-generated file, perhaps template-based, would be a help.

**Comment by alzwded** [comment]
`man finnix` and `dpkg -l`?

If you want to look at what packages get installed when the image gets built, you could snoop through [finnix-live-build](https://github.com/finnix/finnix-live-build), e.g. [lists](https://github.com/finnix/finnix-live-build/tree/main/lists) or [hooks](https://github.com/finnix/finnix-live-build/tree/main/files/hooks) etc

**Reply by alzwded** [reply]
[distrowatch](https://distrowatch.com/table.php?distribution=finnix&pkglist=true&version=125#pkglist) also has a list
